Fix Pylint plugin for Pylint 2.8

SwooshyCueb opened the pull request:
If FlakeHell's pylint plugin fails to import `__pkginfo__.version`, it gives up on pylint completely.
Pylint 2.8 changes `__pkginfo__` quite significantly, and `__pkginfo__.version` has become `__pkginfo__.__version__`.
This fixes the plugin to accommodate for the change.

bergerx commented:
Wouldn't it be better to follow the suggestion in 2.8 release notes:
https://pylint.pycqa.org/en/latest/whatsnew/2.8.html#breaking-changes
> `python from importlib import metadata; metadata.metadata('pylint')` Prefer that to an import from __pkginfo__.

Bahus commented:
Will it be merged sometime?

sgaynetdinov commented:
Hi @SwooshyCueb, thanks for PR

We can use it like this ?
```
import pylint
version = pylint.__version__
```

SwooshyCueb commented:
@bergerx I've updated the PR to try and use importlib, but since importlib wasn't part of the stdlib until 3.8, I've left pulling `__version__` as a fallback.

@sgaynetdinov I've changed the PR to pull `__version__` from `pylint` instead of `pylint.__pkginfo__`.

bergerx commented:
@sgaynetdinov any chance to merge this PR?
